src/components/vertical-divider/vertical-divider.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { testStyledSystemSpacing } from "__spec_helper__/test-utils";
+import VerticalDivider from ".";
+import { MenuContext } from "../menu/menu.component";
+
+testStyledSystemSpacing((props) => <VerticalDivider {...props} />, {
+  p: 3,
+});
+
+test("should render with expected default styles", () => {
+  render(<VerticalDivider />);
+  const verticalDividerElement = screen.getByTestId("divider");
+
+  expect(verticalDividerElement).toHaveStyle({
+    height: "100%",
+    borderLeft: "1px solid #CCD6DB",
+    display: "inherit",
+  });
+});
```

Comment:
**question**: is there any particular reason for including this test? I think that chromatic snapshots should be enough to test this (unless its for coverage).

Reply:
Yet again, I can delete the test and see if it negatively impacts the coverage. If not, we can leave the visual stuff to Chromatic.